src/components/Form/Form.js

```diff
@@ -1,84 +1,36 @@
-import dayjs from 'dayjs';
 import {nanoid} from 'nanoid';
-import {useState} from 'react';
+import dynamic from 'next/dynamic';
+import {useRouter} from 'next/router';
 import styled from 'styled-components';
 
+import useStore from '../../Hooks/useStore';
 import StyledButton from '../Button/StyledButton';
-import DiaryHeadline from '../Headline/DiaryHeadline';
 import Headline2 from '../Headline/Headline2';
 
-const StyledInput = styled.textarea`
-	display: flex;
-	flex-wrap: wrap;
-	width: 400px;
-	height: 100px;
-	margin: 30px 0;
-	padding: 10px;
-	border: 1px solid var(--rose);
-	border-radius: 5px;
-	color: var(--turq);
-`;
-
-const StyledLabel = styled.label`
-	display: none;
-`;
-
-const StyledWrapper = styled.div`
-	display: flex;
-	flex-direction: column-reverse;
-	width: 400px;
-	margin-bottom: 20px;
-	border-radius: 5px;
-	gap: 10px;
-`;
-
-const StyledLi = styled.li`
-	display: flex;
-	flex-direction: column;
-	margin: 0 15px 10px 15px;
-	color: var(--turq_light);
-	gap: 50px;
-`;
-
-const StyledCard = styled.p`
-	display: flex;
-	flex-direction: column-reverse;
-	width: 800px;
-	border: 1px solid var(--turq_light);
-	border-radius: 5px;
-`;
-
-const StyledH4 = styled.h4`
-	display: flex;
-	order: 1;
-	margin-left: 15px;
-	color: var(--turq_light);
-`;
-
-const StyledDate = styled.div`
-	order: 2;
-	margin-left: 15px;
-	color: var(--turq_light);
-`;
-
-function Date({datum}) {
-	return <StyledDate>{dayjs(datum).format('DD.MM.YYYY h:mm A')}</StyledDate>;
-}
+console.clear();
 
 export default function Form() {
-	const [entries, setEntries] = useState([]);
+	const addEntry = useStore(state => state.addEntry);
+	const router = useRouter();
 
 	function handleSubmit(event) {
 		event.preventDefault();
 
 		const formData = new FormData(event.target);
-		const data = Object.fromEntries(formData);
+		const {firstEntry, secondEntry, thirdEntry} = Object.fromEntries(formData);
 
-		setEntries([...entries, {...data, id: nanoid()}]);
 		event.target.reset();
+		const entry = {id: nanoid(), first: firstEntry, second: secondEntry, third: thirdEntry};
+		addEntry(entry);
+		console.log(entry);
+
+		router.push('./diary');
 	}
+	const DynamicWrapper = dynamic(() => import('../styledWrapper'), {
+		ssr: false,
+	});
```

Comment:
The file name should be written in PascalCase.
```suggestion
	const DynamicWrapper = dynamic(() => import('../StyledWrapper'), {
		ssr: false,
	});
```